### ISSUE TITLE
domains that are missing from the a11y domains view

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#569]
* gsa.gov
* ...

**Comment by gbinal** [comment]
This is addressed by a temporary fix but needs to be factored into a longer term solution.

**Comment by gbinal** [comment]
At issue was when there weren't any scan errors.  The updated workflow now accounts for this.
